content/rendering-with-batch/batch/job_definition.md

```diff
@@ -38,6 +38,7 @@ Let's explore the configuration parameters in the structure:
 - **vcpus**: The number of vCPUs reserved for the job. Each vCPU is equivalent to 1,024 CPU shares.
 - **memory**: hard limit (in MiB) for a container. If your container attempts to exceed the specified number, it's terminated.
 - **command**: this is the command that will be executed in the container when the job is started. It has placeholders for some parameters that will be substituted when submitting the job using AWS Batch.
+- **retryStrategy**: this sections defines the retry count for each AWS Batch job. When we simulate an EC2 Spot interruption notice, this retry policy allows AWS Batch to automatically manage the retry.
```

Comment:
Contains orthographic errors. Also, in the way that it's phrased it seems the retry strategy is only used when simulating interruptions. I suggest rephrasing it